python/pysrc/kaskada/_timestream.py

```diff
@@ -1009,6 +1009,51 @@ def cast(self, data_type: pa.DataType) -> Timestream:
         """
         return Timestream(self._ffi_expr.cast(data_type))
 
+    def seconds_since(self, time: Union[Timestream, Literal]) -> Timestream:
+        """
+        Create a Timestream containing the number of seconds since `time`.
```

Comment:
Clarify docs -- what is "now"? Is this "seconds between `time` and the value of `self`? "seconds between `time` and `self.time_of()`, etc.?